.circleci/validate_changed_seed.py

```diff
@@ -137,7 +137,7 @@ def check_gene_alias_table(cursor):
     number_aliases = cursor.fetchone()[0]
     if number_aliases == 0:
         eprint('No genes in seed database')
-    elif number_aliases < 65000:
+    elif number_aliases < 50000:
         # The seed in September 2018 contains 66840 gene aliases.
```

Comment:
@yichaoS good to update the comment as well ☝️ 
```suggestion
        # The seed in September 2018 contains ?? gene aliases.
```

Reply:
@inodb updated this comment with the latest number.